templates/js/linker.v2.js

```diff
@@ -409,32 +411,37 @@
             var book = books[k];
             // Run each regex over the document, and wrap results
             var r = XRegExp(ns.regexes[book],"xgm");
+            // find the refrences and push them into ns.matches
             for (var i = 0; i < ns.elems.length; i++) {
                 findAndReplaceDOMText(ns.elems[i], {
                     preset: 'prose',
                     find: r,
-                    replace: function(portion, match) {
+                    replace: (function(book, portion, match) {
                         var matched_ref = match[0]
                             .replace(/[\r\n\t ]+/g, " ") // Filter out multiple spaces
-                            .replace(/[(){}[\]]+/g, ""); // Filter out internal parenthesis todo: Don't break on parens in books names
-                        ns.matches.push(matched_ref);
-
-                        var node = document.createElement("a");
-                        node.target = "_blank";
-                        node.className = "sefaria-ref";
-                        node.href = base_url + matched_ref;
-                        node.setAttribute('data-ref', matched_ref);
-                        node.setAttribute('aria-controls', 'sefaria-popup');
-                        node.textContent = portion.text;
-
-                        return node;
-                    },
+                            .replace(/[(){}[\]]+/g, "")
+                            .replace('\u05d3\u05e3', ""); // Filter out internal parenthesis todo: Don't break on parens in books names
+                            var matched_reg = new RegExp(`${book}\\s+([\u05d0-\u05ea]+?['\u05f3"\u05f4][\u05d0-\u05ea]*?(:|\\.)?(\\s|$)|[\u05d0-\u05ea](\\.|:)?([-\u2010-\u2015\u05be][\u05d0-\u05ea])?(\\s|$)|(\\d|\\.)+(a|b|:)?(\\s|[-\u2010-\u2015\u05be]|$))+`, 'g');
```

Comment:
these next few lines need comments

Reply:
Also, does this catch ranges in English?

Reply:
yes. It does catch English ranges, given that the ranges are in numbers (ex Job 3.1-2)